sdk/nodejs/sdk/src/zrok/api/.openapi-generator/FILES

```diff
@@ -1,5 +1,4 @@
 .gitignore
-.openapi-generator-ignore
```

Comment:
why are we touching stuff inside the nodejs tree?

Reply:
It's not intentional and shows that generating on my machine produces a slightly different result. I'll look for a way to configure that generator so as not to introduce this change.

Reply:
Let's not change the generation. Let's just focus on adjusting the python SDK to properly use the new generated code. I really do not want to introduce generation changes right now.

Reply:
And the critical issue is figuring out how to structure our python SDK so that the generated output lives in a separate folder from the code we vend around it.